app/Http/Controllers/RealopsController.php

```diff
@@ -220,6 +220,22 @@ public function deleteFlight($id) {
         return redirect()->back()->with('success', 'That flight has been deleted successfully');
     }
 
+    public function dumpData(Request $request) {
+        if ($request->input('confirm_text') != "confirm - dump all") {
```

Comment:
I recommend that you add a strtolower() in here on the confirm text test... if someone actually typed "confirm - dump all" then we know that they want to do it... let's not chance penalizing them for typing it in camelcase or uppercase.

Reply:
Resolved and squashed into 0c3d3ce